### PR TITLE
fix #140

### PR DESCRIPTION
**Pull request by ven-kyoshiro**
fix #140 and done run_diayn,py test in my computer.
Please check!

### CODE REVIEW

File: machina/traj/epi_functional.py [hunk 1]
```diff
@@ -207,6 +207,16 @@ def compute_pseudo_rews(data, rew_giver, state_only=False):
     return data
 
 
+def compute_diayn_rews(data, discrim):
+    epis = data.current_epis
+    for epi in epis:
+        obs = torch.tensor(epi['obs'], dtype=torch.float, device=get_device())
+        with torch.no_grad():
+            rews, info = discrim(obs)
```
Comment (rarilurelo) [inline machina/traj/epi_functional.py]: Is this `discrim` wrapped by calc_rewards?

**Comment by rarilurelo** [comment]
Could you add test for DIAYN?

File: machina/traj/epi_functional.py (hunk 1 above)
Comment (ven-kyoshiro) [reply machina/traj/epi_functional.py]: done by wrapped at run_diayn,py
https://github.com/DeepX-inc/machina/pull/170/commits/cd539a0367cac47592185fe9ecdf88a5703e394f

**Comment by ven-kyoshiro** [comment]
Done adding a test of DIAYN.
Please check 👍

**Pull request merged**